vendor/github.com/machadovilaca/operator-observability/pkg/operatorrules/prometheusrules.go

```diff
@@ -62,6 +62,7 @@ func buildRecordingRulesRules() []promv1.Rule {
 		rules = append(rules, promv1.Rule{
 			Record: recordingRule.MetricsOpts.Name,
 			Expr:   recordingRule.Expr,
+			Labels: recordingRule.MetricsOpts.ConstLabels,
```

Comment:
@avlitman Do you still need this?

Reply:
It's not needed, but nice to have since it is latest version.